src/routes/entry.js

```diff
@@ -52,10 +63,66 @@ module.exports = function (hds) {
         this.body = entry;
     }
 
-    function*getAttachment() {
+    function* createEntry() {
+        var data = this.request.body;
+        try {
+            var value = yield hds.Entry.create(this.params.kind, data, {owner: 'test@cheminfo.org'}).save();
+            this.body = {
+                status: 'created',
+                entryID: value._id
+            };
+        } catch(err) {
+            this.hds_jsonError(500, err);
+        }
+    }
+
+    function* deleteEntry() {
+        try {
+            yield this.state.hds_entry.remove();
+            this.body = {status: 'deleted'};
+        } catch (err) {
+            this.hds_jsonError(500, err);
+        }
+    }
+
+    function* changeEntry() {
+        try {
+            yield hds.Entry.update({_id: this.state.hds_entry._id}, { $set: this.request.body});
+            this.body = {status: 'modified'};
+        } catch (err) {
+            this.hds_jsonError(500, err);
+        }
+    }
+
+    function* queryEntry() {
+        try {
+            var data = this.request.body;
+            var from = data.from || 0;
+            var limit = data.limit || 20;
+            var entries = yield this.state.hds_kind
+                .find(data.query)
+                .skip(from)
+                .limit(limit - from)
```

Comment:
checking the doc I'm still not sure if this is the expected value for `.limit()`